packages/functional-tests/tests/cms/cms.spec.ts

```diff
@@ -12,413 +12,438 @@ const ENTRYPOINT_SYNC = 'firefox-cms';
+      }: {
+        headline: string;
+        description: string;
+        logoUrl?: string;
+        logoAlt?: string;
+        buttonColor: string;
+        buttonText: string;
+      }
+    ) {
+      await expect(page.getByRole('heading', { name: headline })).toBeVisible();
       await expect(page.getByText(description)).toBeVisible();
-    }
 
-    if (logoUrl) {
-      const logo = page.getByRole('img', { name: logoAlt, exact: true });
-      await expect(logo).toBeVisible();
-      await expect(logo).toHaveAttribute('src', logoUrl);
-    }
+      if (description) {
+        await expect(page.getByText(description)).toBeVisible();
+      }
 
-    const button = page.getByRole('button', { name: buttonText, exact: true });
-    await expect(button).toBeVisible();
-    await expect(button).toHaveClass(/cta-primary-cms/);
-    await expect(button).toHaveClass(/cta-xl/);
-    await expect(button).toHaveClass(/cta-primary/);
-    await expect(button).toHaveCSS('--cta-bg', buttonColor);
-    await expect(button).toHaveCSS('--cta-border', buttonColor);
-    await expect(button).toHaveCSS('--cta-active', buttonColor);
-    await expect(button).toHaveCSS('--cta-disabled', `${buttonColor}60`);
-  }
-  test.beforeAll(async ({ target}) => {
-    // These tests require customizations for 123Done and Sync. Please contact
-    // account developer to get access to Strapi dev instance.
-    const relierConfig = await target.authClient.getCmsConfig(CLIENTID_123Done, ENTRYPOINT_123Done);
-    const syncConfig = await target.authClient.getCmsConfig(CLIENTID_SYNC, ENTRYPOINT_SYNC);
-
-    // Check if configs are empty or null/undefined
-    const isConfigEmpty = (config:any ):boolean => {
-      return (
-        !config ||
-        typeof config !== 'object' ||
-        Object.keys(config).length === 0
-      );
-    };
+      if (logoUrl) {
+        const logo = page.getByRole('img', { name: logoAlt, exact: true });
+        await expect(logo).toBeVisible();
+        await expect(logo).toHaveAttribute('src', logoUrl);
+      }
 
-    test.skip(isConfigEmpty(relierConfig) || isConfigEmpty(syncConfig), 'CMS customization is not set up for 123Done or Sync');
-  });
-
-  test.beforeEach(async ({ pages: { configPage } }) => {
-    // Ensure that cms customization is enabled
-    const config = await configPage.getConfig();
-    test.skip(config.cms.enabled !== true, 'CMS customization is not enabled');
-  });
-
-  test.describe('Relying Party customization', () => {
-    test('default experience with invalid entrypoint', async ({
-      target,
-      page,
-      pages: { relier },
-      testAccountTracker,
-    }) => {
-      await testAccountTracker.generateAccountDetails();
-
-      await relier.goto('entrypoint=invalid');
-      await relier.clickEmailFirst();
-
-      await expect(
-        page.getByRole('heading', { name: 'Enter your email' })
-      ).toBeVisible();
-      await expect(page.getByText(/Continue to .*123Done/)).toBeVisible();
```

Comment:
This is the main change, everything else is indention. The assert is problematic and probably a footgun since its pulling from the oauth database.